README.md

```diff
@@ -1,8 +1,8 @@
 # WP SAML Auth #
-**Contributors:** [getpantheon](https://profiles.wordpress.org/getpantheon/), [danielbachhuber](https://profiles.wordpress.org/danielbachhuber/), [outlandish-josh](https://profiles.wordpress.org/outlandish-josh/), [jazzs3quence](https://profiles.wordpress.org/jazzs3quence/)  
+**Contributors:** [getpantheon](https://profiles.wordpress.org/getpantheon/), [danielbachhuber](https://profiles.wordpress.org/danielbachhuber/), [outlandish-josh](https://profiles.wordpress.org/outlandish-josh/), [jazzs3quence](https://profiles.wordpress.org/jazzs3quence/), [lcatlett](https://profiles.wordpress.org/lcatlett/)  
 **Tags:** authentication, SAML  
 **Requires at least:** 4.4  
-**Tested up to:** 6.3  
+**Tested up to:** 6.4.3  
 **Requires PHP:** 7.3  
 **Stable tag:** 2.1.5-dev  
```

Comment:
```suggestion
**Stable tag:** 2.2.0-dev  
```
bump this to a minor release as part of this same PR (search-replace across all the places that matter)

Reply:
I was planning on doing that anyway. Also, I will likely update the fixture and change the tested-up-to to 6.8.1, I just haven't gotten to it yet.

Reply:
Bumped stable tag to 2.2.0-dev and tested-up-to to 6.8.1